EKSD_LATEST_RELEASES

```diff
@@ -1,38 +1,16 @@
-releases:
```

Comment:
Decided to clean this up as I didn't see any reason to maintain the deprecated versions.

Reply:
i think we may a couple of them, for the previous pause image tag logic. id revert this honestly and do this separately if at all